pkg/libcose/Makefile.dep

```diff
@@ -8,3 +8,9 @@ endif
 ifneq (,$(filter libcose_crypt_hacl,$(USEMODULE)))
   USEPKG += hacl
 endif
+ifneq (,$(filter libcose_crypt_monocypher,$(USEMODULE)))
```

Comment:
@bergzand are `monocypher` and `c25519` still missing RIOT glue code? If its the case I might take them out of this PR for now.

Reply:
@kaspar030 testes #11818 using `moncypher` and `c25519` and they are not working yet.

I order not to delay I would suggest removing them from the PR, and going forward with the rest.

Reply:
+1, I've removed the corresponding commit